classes/dojo.py

```diff
@@ -0,0 +1,133 @@
+from classes.office import Office
+from classes.living_space import LivingSpace
+from classes.fellow import Fellow
+from classes.staff import Staff
+from random import randint
+
+
+class Dojo(object):
+    """
+    This class imports Office, LivingSpace, Fellow and Staff.
+
+    It creates instances of each in its methods.
+    """
+
+    def __init__(self):
+        self.all_rooms = []
+        self.allocated_persons = []
+        self.unallocated_persons = []
+        self.all_persons = []
+        self.offices = []
+        self.living_spaces = []
+
+    def create_room(self, room_type, room_name):
+        if not isinstance(room_name, str) or not isinstance(room_type, str):
+            raise ValueError('Invalid: Please input a string as room_name or room_type')
+        else:
+            room_type = room_type.lower()
+            room_name = room_name.lower()
+        if room_type != "office" and room_type != "living_space":
+            raise ValueError("Invalid room type: Your room type should be 'office' or 'living_space'")
+        existing_rooms = []
+        for room in self.all_rooms:
```

Comment:
Why not just check against the `self.all_rooms` list, instead of creating a new list that only exists within the  `create_room()` function?